components/crud-web-apps/jupyter/frontend/src/polyfills.ts

```diff
@@ -1,3 +1,7 @@
+/***************************************************************************************************
+ * Load `$localize` onto the global scope - used if i18n tags appear in Angular templates.
+ */
+import '@angular/localize/init';
```

Comment:
Hello @DavidSpek and thanks for your PR. I noticed that you added this package in all UIs. Can I ask why?

Reply:
That is needed for the i18n as of Angular 9. It stated this in the upgrade documentation.

Reply:
I don't see any usage of i18n in this branch, so it shouldn't be necessary. But it won't hurt either. Let's leave it there since Angular will not include it in the bundle and we might need it in the future.

Reply:
The i18n for the web apps was merged before I did the upgrade process, so that’s why I added the package. I actually needed to do the upgrade twice since my first branch accidentally missed the latest commits that included the i18n which resulted in merge conflicts with master.